build/csv/ceph/rook-ceph-operator.clusterserviceversion.yaml

```diff
@@ -3052,6 +3052,11 @@ spec:
                           configMapKeyRef:
                             key: ROOK_CURRENT_NAMESPACE_ONLY
                             name: ocs-operator-config
+                      - name: ROOK_CSI_ENABLE_CEPHFS
+                        valueFrom:
+                          configMapKeyRef:
+                            key: ROOK_CSI_ENABLE_CEPHFS
+                            name: ocs-operator-config
```

Comment:
It seems odd to mount the ocs operator config to the rook operator. Instead of mounting that configmap, what if we just set the env value directly to enabled it by default? Then the user can override it with the configmap that rook already checks. 
```yaml
                    - name: ROOK_CSI_ENABLE_CEPHFS
                      value: "true"
```

Reply:
So in the case of ocs-operator deploying rook, how user automation can set the value in the rook cm? 
We thought of adding it in the ocs operator configmap and then to the rook operator deployment env  varibles

Reply:
The same pattern we follow for `ROOK_CURRENT_NAMESPACE_ONLY`

Reply:
Ok, thanks for the discussion, I see now that this is needed to pick up the downstream settings from ocs operator, and follows the same pattern of some other settings.